src/arutils.py

```diff
@@ -437,6 +456,8 @@ def polyval2d(x, y, m):
         z += a * x**i * y**j
     return z
 
+def gauss_3deg(x,ampl,cent,sigm):
+    return ampl*np.exp(-1*(cent-x)**2/2/sigm**2)
```

Comment:
Let's discuss.   At the S/N we should tend to have, I suspect this
is a non-issue, but I could be persuaded otherwise.